src/app/model/GameRecord.ts

```diff
@@ -0,0 +1,9 @@
+import { Piece } from './pieces/Piece';
+import { Side, Square } from './Types';
+
+export type GameRecord = {
```

Comment:
Jeśli ten typ jest używany tylko w jednym pliku, a szczególnie w jednej klasie to dobrze zostawić go jako private w tej klasie :) Wtedy to się staje szczegółem implementacyjnym, a nie czymś z czego ktoś może inny skorzysta i powiąże ściśle dwie części kodu.

Reply:
Ja to w ogóle wywalę

Reply:
done